tools/ocean_neural_network/canyonb.xml

```diff
@@ -0,0 +1,91 @@
+<tool id="bgc_canyon_b" name="Canyon-B" version="@VERSION@">
+    <description>Robust Estimation of Open Ocean CO2 Variables and Nutrient Concentrations From T, S, and O2 Data Using Bayesian Neural Networks</description>
+    <macros>
+        <token name="@VERSION@">0.9.3</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="4.3.2">r-base</requirement>
+        <requirement type="package" version="@VERSION@">r-canyonb</requirement>
+    </requirements>
+    <required_files>
+        <include type="literal" path="canyonb.r"/>
```

Comment:
What is type literal?

Reply:
This something I remembered you ask me to add in my tools a while back. But I don't remember what literal means I just remember it's to have a trace of the files needed for the tool to work.

Reply:
https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-required-files-include

tpye="path" isn't it?

Reply:
Is the galaxy language server complaining?

Reply:
all good

Reply:
`.. ERROR (XSD): Invalid XML: Element 'include', attribute 'type': [facet 'enumeration'] The value 'path' is not an element of the set {'literal', 'prefix', 'glob', 'regex'}.`

Reply:
I only know https://github.com/galaxyproject/tools-iuc/blob/4dba8c02f5b33b425d4ecd9c9f0d0eaa0a74f9bf/tools/extract_genomic_dna/extract_genomic_dna.xml#L8 to be honest.

But you are more correct here. TIL. Thanks!